GOVERNANCE.md

```diff
@@ -101,21 +93,20 @@ Advisory Committee members:
 In the event of any conflict of interest (a Committee Member, their family member, or someone with whom the Committee Member has a close academic or employment relationship is involved in a decision), the Committee Member must immediately notify other Committee Members. The Committee Member will be asked to recuse themselves from ongoing conversations, and decision process regarding the Transaction.  
 
 #### Current list of Advisory Committee members
-* Martin Magdinier
 * Jan Ainali
 * Antonin Delpeuch
+* [Open Position]
 
-### Project Director
-OpenRefine's Project Director (paid position) works closely with OpenRefine’s advisory and steering committees, and with its communities of users and contributors.
+### Project Manager
+OpenRefine's Project Manager (paid position) works closely with OpenRefine’s advisory and steering committees, and with its communities of users and contributors.
 
-The Project Director:
+The Project Manager:
 * Helps to improve OpenRefine's governance and community diversity
 * Helps build community-driven structures to formalize OpenRefine’s roadmap and keep it up to date
 * Assists in finding and secure new sources of funding
 * Supports the Advisory Committee in OpenRefine’s day-to-day operations.
 
-Currently OpenRefine has no project director
-
+The current Release Manager is [Martin Magdinier](https://forum.openrefine.org/u/Martin/summary)
```

Comment:
"Release Manager"-> "Project Manager"?

Reply:
thanks for catching this. It is two different role. I made the correction